packages/mobile/src/screens/sign-on-screen/screens/SelectArtistScreen/FollowArtistField.tsx

```diff
@@ -0,0 +1,108 @@
+import type { UserMetadata } from '@audius/common'
+import { css } from '@emotion/native'
+import { useField } from 'formik'
+
+import {
+  Box,
+  Flex,
+  FollowButton,
+  IconNote,
+  IconUser,
+  Paper,
+  Text,
+  useTheme
+} from '@audius/harmony-native'
+import {
+  Divider,
+  ProfilePicture,
+  UserCoverPhoto,
+  UserDisplayName
+} from 'app/components/core'
+import { StaticSkeleton } from 'app/components/skeleton'
+
+type FollowArtistFieldProps = {
+  artist: UserMetadata
+}
+
+export const FollowArtistField = (props: FollowArtistFieldProps) => {
+  const { artist } = props
+  const { user_id, track_count, follower_count } = artist
+  const { spacing } = useTheme()
+  const [{ onChange }] = useField({ name: 'selectedArtists', type: 'checkbox' })
```

Comment:
@DejayJD was able to figure out how to do the checkbox style on mobile. im also seeing per issues though, so will need to follow up with something similar to what we did with select-genre.